webpack.common.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const _ = require('underscore');
+const pkg = require('./package.json');
+
+module.exports = {
+    mode: 'production',
+    entry: './index.js',
+    resolve: {
+        extensions: ['.jsx', '.js'],
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules)/,
+                use: {
+                    loader: 'babel-loader',
+                },
+            },
+        ],
+    },
+    externals: [
+        'react-native',
+        /^expensify-common\/.+$/,
+        /^lodash\/.+$/,
```

Comment:
Regex syntax is intentional since we import subpaths of these modules 
- `lodash/merge` 
- `expensify-common/str/...` 

If we don't use a pattern the whole package is bundled even if it's a peer dependency: https://webpack.js.org/guides/author-libraries/#external-limitations